src/features/MentorPage/ListCard/ListCardHeader.tsx

```diff
@@ -0,0 +1,152 @@
+import styled from 'styled-components';
+import * as cssVariables from '../../../components/variables';
+import ProfilePicPlaceholder from '../../../static/img/icon-chat-profilepic.svg';
+import CSS from 'csstype';
+
+type Props = {
+  name: string;
+  age: number;
+  region: string;
+  available: boolean;
+  newMentor: boolean;
+  message: string;
+};
+
+const customStylus: CSS.Properties = {
+  margin: '0px',
+  display: 'flex',
+  flexWrap: 'wrap',
+};
+
+const truncateText: CSS.Properties = {
+  textOverflow: 'ellipsis',
+  whiteSpace: 'nowrap',
+  overflow: 'hidden',
+  margin: '0px',
+  width: '100%',
+};
+
+const ListCardHeader: React.FC<Props> = ({
+  name,
+  age,
+  region,
+  available,
+  newMentor,
+  message,
+}) => {
+  let availabilityMessage = newMentor ? 'Uusi' : '';
+  availabilityMessage = available ? availabilityMessage : 'Ei tavoitettavissa';
```

Comment:
Here we reassign the variable, big no-no from me!

Instead we could have two variables
```suggestion
   const newCardMessage = isNewMentor ? 'Uusi' : '';
   const availabilityMessage = isAvailable ? newCardMessage : 'Ei tavoitettavissa';
```